### ISSUE TITLE
[14.0] web_search_with_and: terms are added twice when pressing shift+enter

### ISSUE THREAD

**Issue by alessandro-fiorino** [OCA/web#2488]
## Module
web_search_with_and

## Describe the bug
Terms are added twice when pressing shift+enter

## To Reproduce
**Affected versions**: 14.0

1. go in any search omnibar, i.e. in contacs
2. type a term and then press shift+enter
3. the term is added twice 

**Expected behavior**
The term should be added only once

**Comment by alessandro-fiorino** [comment]
Duplicate of #2291
